libs/utils/utils.js

```diff
@@ -603,8 +603,20 @@ export async function loadDeferred(area, blocks, config) {
 
   if (config.locale?.ietf === 'ja-JP') {
     // Japanese word-wrap
-    import('../features/japanese-word-wrap.js').then(({ controlLineBreaksJapanese }) => {
-      controlLineBreaksJapanese(config, area);
+    import('../features/japanese-word-wrap.js').then(({ default: controlLineBreaksJapanese }) => {
+      const budouxSelector = getMetadata('jpwordwrap:budoux-selector') || 'h1, h2, h3, h4, h5, h6';
```

Comment:
Can this fn be moved into japanese-word-wrap.js?

Reply:
Okay!